src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -121,6 +144,27 @@ namespace chip {
 #define CHIP_CONFIG_RMP_DEFAULT_MAX_RETRANS (4)
 #endif // CHIP_CONFIG_RMP_DEFAULT_MAX_RETRANS
 
+/**
+ *  @def CHIP_CONFIG_MRP_RETRY_INTERVAL_SENDER_BOOST
+ *
+ *  @brief
+ *    A constant value that should be added to the calculated retransmission
+ *    timeout when the present node tramsmits a message.
+ *
+ *  The purpose for this constant is to limit the possibility of spurious
+ *  rertansmissions in the scenario in which a sender that operates in a high-
+ *  latency network (such as Thread) sends a message to a receiver that operates
+ *  in a low-latency network (such as Wi-Fi). In this scenario, the SAI and SII
+ *  parameters advertised by the receiver are low although the average round-
+ *  trip time of a big request-response pair is long due to the nature of the
+ *  sender.
+ */
+#ifndef CHIP_CONFIG_MRP_RETRY_INTERVAL_SENDER_BOOST
+#if CHIP_ENABLE_OPENTHREAD
+#define CHIP_CONFIG_MRP_RETRY_INTERVAL_SENDER_BOOST (500_ms)
```

Comment:
Just a question.. "ms" or "ms32" as above?

Reply:
Do i correctly understand that 500ms was calculated as 800ms - 300ms?

Reply:
Right, should be `_ms32`. It is cast, anyway, but that is more correct. Yeah, I wanted to make sure that the retransmission timeout is not lower than 800ms.

I'll contribute the changes to master, but our upstream tests are not functional at the moment so I need some confirmation that these values to trigger any other issues (which I can only get from NCS CI).